pom.xml

```diff
@@ -28,8 +28,20 @@
 
 	<dependencyManagement>
 		<dependencies>
-				<dependency><groupId>org.openmrs.contrib</groupId><artifactId>testutils</artifactId><version>${openmrsTestutilsVersion}</version><scope>test</scope></dependency>
-
+			<dependency><groupId>org.openmrs.contrib</groupId><artifactId>testutils</artifactId><version>${openmrsTestutilsVersion}</version><scope>test</scope></dependency>
+			<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>webservices.rest-omod</artifactId>
+				<version>${webServicesRestVersion}</version>
+				<scope>provided</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.openmrs.module</groupId>
+				<artifactId>webservices.rest-omod-common</artifactId>
+				<version>${webServicesRestVersion}</version>
+				<classifier>tests</classifier>
```

Comment:
i already did. i need them on openmrs-web though. for that dependency i tried adding test-jar, but without the classifier, the tests fail. @wluyima